wolfssl/wolfio.h

```diff
@@ -69,11 +69,12 @@
     #elif defined(FREESCALE_KSDK_MQX)
         #include <rtcs.h>
     #elif (defined(WOLFSSL_MDK_ARM) || defined(WOLFSSL_KEIL_TCP_NET))
-        #include "cmsis_os.h"
```

Comment:
Will removing this break existing users builds?

Reply:
Rollbacked